mantidproject/mantid
# Workbench sample logs: Use Run::startTime to compute t0

**Description of work.**

Uses the recorded start time in the `run` to determine where `t=0` should be plotted on the sample log plots.

**Report to:** Adrian Hillier, ISIS

**To test:**

Best to be tested at ISIS using the muon file shown in the associated issue. Time zero should now occur where at ~1800s from the start of the log times.

Also test and SNS file, e.g. CNCS, ARCS, EQSANS, from system test data and compare the plots from the current nightly. They should be the same as those logs start from run start.

Fixes #26222 

*This does not require release notes* because **the sample log viewer was released this release cycle.**

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/IndividualTicketTesting/)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
